radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/core/api.yaml

```diff
@@ -0,0 +1,3520 @@
+openapi: 3.1.0
```

Comment:
This not necessarily needs to be done in this PR, but this file is way too big. It is not that trivial to split, but it is doable. Could you add a task for that?